app/controllers/detail-viewers.ts

```diff
@@ -296,8 +302,9 @@ function getTimeExtensionViewer(req: Request, res: Response, next: NextFunction)
   try {
     const timeExtensionId = req.params.id;
     const timeExtension: Collection<Application> = req.session.appeal.makeAnApplications.find(application => application.id === timeExtensionId);
+    const timeExtensionEvidence: Evidence[] = req.session.appeal.makeAnApplicationEvidence;
     const previousPage: string = paths.common.overview;
-    const { request, response = null } = getTimeExtensionSummaryRows(timeExtension);
+    const { request, response = null } = getTimeExtensionSummaryRows(timeExtension, timeExtensionEvidence);
```

Comment:
timeExtension contains the response from CCD and also contains the evidences if any. The object `makeAnApplicationEvidence` is only used to make the request not to read anything from it